libcloud/pricing.py

```diff
@@ -200,6 +202,71 @@ def get_size_price(driver_type, driver_name, size_id, region=None):
     return price
 
 
+def get_gce_image_price(image_name, size_name, cores=1):
```

Comment:
Would it be possible to make this "private" and call it inside ``get_size_price()`` method when GCE pricing is requested?

Since the whole idea behind Libcloud is to try to use common / standard API as much as possible. If we create a provider specific methods everywhere it kinda defeats the purpose (yes, I know it's not always possible to do it in common / standard way, but we should try out best).